JS.md

```diff
@@ -303,16 +303,20 @@ For each [`import`](https://github.com/WebAssembly/spec/blob/master/interpreter/
     1. Let `closure` be `v.[[Closure]]`.
   1. Otherwise:
     1. Let `closure` be a new [host function](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/instance.ml#L9)
-       of the given signature
-       which calls `v` by coercing WebAssembly arguments to JavaScript arguments
-       via [`ToJSValue`](#tojsvalue) and returns the result, if any, by coercing
-       via [`ToWebAssemblyValue`](#towebassemblyvalue).
+       of the given signature:
+      1. If the signature contains an `i64` (as argument or result), the host
+         function immediately throws a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
+         when called.
+      1. Otherwise, the host function calls `v` with an `undefined` receiver
+         and WebAssembly arguments coerced to JavaScript arguments
+         via [`ToJSValue`](#tojsvalue). The result is returned by coercing
+         via [`ToWebAssemblyValue`](#towebassemblyvalue).
   1. Append `v` to `funcs`.
   1. Append `closure` to `imports`.
 1. If `i` is a global import:
   1. [Assert](https://tc39.github.io/ecma262/#assert): the global is immutable
      by MVP validation constraint.
-  1. If `Type(v)` is not Number, throw a `WebAssembly.LinkError`.
+  1. If `v` is an `i64` or `Type(v)` is not Number, throw a `WebAssembly.LinkError`.
```

Comment:
That's actually how we started out and the requirement was tightened by #816 after a bit of [discussion](https://github.com/WebAssembly/spec/issues/356#issuecomment-252851799) in [spec/#356](https://github.com/WebAssembly/spec/issues/356).